src/dragonball/src/device_manager/legacy.rs

```diff
@@ -140,6 +146,83 @@ pub(crate) mod x86_64 {
     }
 }
 
+#[cfg(target_arch = "aarch64")]
+pub(crate) mod aarch64 {
+    use super::*;
+    use dbs_device::device_manager::{IoManager};
+    use dbs_device::resources::DeviceResources;
+    use std::collections::HashMap;
+    use kvm_ioctls::VmFd;
+
+    type Result<T> = ::std::result::Result<T, Error>;
+
+    impl LegacyDeviceManager {
+        pub fn create_manager(
+            bus: &mut IoManager,
+            vm_fd: Option<Arc<VmFd>>,
+            resources: &HashMap<String, DeviceResources>,
+        ) -> Result<Self> {
+            let (com1_device, com1_eventfd) =
+                Self::create_com_device(bus, vm_fd.as_ref(), resources.get("com1").unwrap())?;
+            let (com2_device, com2_eventfd) =
+                Self::create_com_device(bus, vm_fd.as_ref(), resources.get("com2").unwrap())?;
+            let (rtc_device, rtc_eventfd) =
+                Self::create_rtc_device(bus, vm_fd.as_ref(), resources.get("rtc").unwrap())?;
```

Comment:
Nit: It might be worth defining `com1`, `com2` and `rtc` as `public const` values as these strings are duplicated in two files in this PR.

Reply:
Thanks, @jodh-intel ! I have addressed that.